src/vuex/getters.js

```diff
@@ -21,6 +26,22 @@ export const allTransactions = state =>
     state.transactions.governance,
     state.transactions.distribution
   )
+
+export const flatOrderedTransactionList = (state, getters) => {
+  let allTx = getters.allTransactions.reduce(flattenTransactionMsgs, [])
+  allTx = allTx.map(addTransactionTypeData(state))
+  allTx.sort(compareBlockTimeDesc)
+  return allTx
+}
+
+export const validators = state => {
```

Comment:
should `yourValidatorsMap` also be a map in getters?

Reply:
It could be, but this was specific to here.
Thre are more changes I can make there, but it wasstarting to get outside thescope of this PR.